src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1956,6 +1961,14 @@ def get_crash_data(crash_data, symbolize_flag=True):
         PYTHON_STACK_FRAME_FUNCTION_REGEX, line, state, group=3):
       continue
 
+  # Only get repo.prop if we have and Android kernel or KASAN crash
+  if environment.platform() in ('ANDROID', 'ANDROID_KERNEL') and (
+      found_android_kernel_crash or is_kasan):
+    repo_prop_path = settings.get_repo_prop_path()
+    if os.path.exists(repo_prop_path):
+      state.android_kernel_repo = utils.read_data_from_file(
```

Comment:
Where is android_kernel_repo used ?

Reply:
It will be used in the second CL to build the srcmap.